From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 19:59:05 +0200
Subject: Add description to the pacemaker_remote RA for proper man page
 generation

This gets rid of the manpage-has-bad-whatis-entry Lintian warning.
---
 extra/resources/remote |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/extra/resources/remote b/extra/resources/remote
index d79c4c3..197b21a 100644
--- a/extra/resources/remote
+++ b/extra/resources/remote
@@ -45,6 +45,15 @@ meta_data() {
 <!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
 <resource-agent name="remote" version="0.1">
   <version>0.1</version>
+  <longdesc lang="en">
+This resource agent provides metadata for the internal
+pacemaker remote lrmd connection agent.  Outside of acting
+as a place holder so the remote RA script can be indexed and
+providing metadata, this script should never be invoked.  The
+actual functionality behind the remote lrmd connection lives
+within pacemaker's crmd component.
+  </longdesc>
+  <shortdesc lang="en">Provides metadata for the remote connection agent</shortdesc>
   <parameters>
     <parameter name="server" unique="1">
     <longdesc lang="en">
